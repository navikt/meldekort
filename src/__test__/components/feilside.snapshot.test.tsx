import * as React from 'react';
import * as renderer from 'react-test-renderer';
import ProviderWrapper, {
  setLocalesBeforeAll,
} from '../testSetup/providerWrapper';
import Feilside from '../../app/components/feilside/feilside';

setLocalesBeforeAll();

it('Feilside', () => {
  const tree = renderer.create(
    <ProviderWrapper>
      <Feilside />
    </ProviderWrapper>
  );
  expect(tree).toMatchSnapshot();
});
