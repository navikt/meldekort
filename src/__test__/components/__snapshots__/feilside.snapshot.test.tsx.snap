// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feilside 1`] = `
<main
  className="sideinnhold"
>
  <h1
    className="typo-innholdstittel seksjon flex-innhold sentrert"
  >
    <span
      dangerouslySetInnerHTML={
        Object {
          "__html": "Du har ikke tilgang til meldekort ",
        }
      }
    />
  </h1>
  <section
    className="seksjon flex-innhold sentrert"
  >
    <div
      className="navds-alert navds-alert--error navds-alert--medium"
    >
      <svg
        aria-labelledby="title-"
        className="navds-alert__icon"
        fill="none"
        focusable={false}
        height="1em"
        role="img"
        viewBox="0 0 24 24"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <title
          id="title-"
        >
          Feil
        </title>
        <path
          clipRule="evenodd"
          d="M7.742 2.47a.75.75 0 0 1 .53-.22h7.456a.75.75 0 0 1 .53.22l5.272 5.272c.141.14.22.331.22.53v7.456a.75.75 0 0 1-.22.53l-5.272 5.272a.75.75 0 0 1-.53.22H8.272a.75.75 0 0 1-.53-.22L2.47 16.258a.75.75 0 0 1-.22-.53V8.272a.75.75 0 0 1 .22-.53L7.742 2.47Zm1.288 5.5a.75.75 0 0 0-1.06 1.06L10.94 12l-2.97 2.97a.75.75 0 1 0 1.06 1.06L12 13.06l2.97 2.97a.75.75 0 1 0 1.06-1.06L13.06 12l2.97-2.97a.75.75 0 0 0-1.06-1.06L12 10.94 9.03 7.97Z"
          fill="currentColor"
          fillRule="evenodd"
        />
      </svg>
      <div
        className="navds-alert__wrapper navds-body-long navds-body-long--medium"
      >
        <span
          dangerouslySetInnerHTML={
            Object {
              "__html": "For å være registrert som arbeidssøker eller for å få dagpenger, AAP og tiltakspenger, må du sende meldekort hver 14. dag. Logg på nav.no og velg \\"Registrer deg som arbeidssøker\\", hvis du vil registrere deg. ",
            }
          }
        />
      </div>
    </div>
  </section>
</main>
`;
